docs/en/pid-eaa-entity-configuration.rst

```diff
@@ -0,0 +1,141 @@
+    - JSON array containing a list of Proof Key for Code Exchange (PKCE) :rfc:`7636` code challenge methods supported by the authorization server. The authorization server MUST support *S256*.
+  * - **acr_values_supported**
+    - See `OpenID Connect Discovery 1.0 Section 3 <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata>`_. The supported values are:
+      
+      - `https://www.spid.gov.it/SpidL1`
+      - `https://www.spid.gov.it/SpidL2`
+      - `https://www.spid.gov.it/SpidL3`
+  * - **scopes_supported**
+    - JSON array containing a list of the supported *scope* values. See :rfc:`8414#section-2`.
+  * - **response_modes_supported**
+    - JSON array containing a list of the supported "response_mode" values, as specified in `OAuth 2.0 Multiple Response Type Encoding Practices <https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html>`_. The supported values MAY be *query* and *form_post.jwt* (see `[oauth-v2-jarm-03] <https://openid.net/specs/oauth-v2-jarm-03.html#name-response-mode-form_postjwt>`_).
+  * - **authorization_signing_alg_values_supported**
+    - JSON array containing a list of the JWS :rfc:`7515` supported signing algorithms (*alg* values). The values MUST be set according to Section :ref:`Cryptographic algorithms`. See Section 4 of `[oauth-v2-jarm-03] <https://openid.net/specs/oauth-v2-jarm-03.html>`_.
+  * - **grant_types_supported**
+    - JSON array containing a list of the supported grant type values. The authorization server MUST support *authorization_code*. 
+  * - **token_endpoint_auth_methods_supported**
+    - JSON array containing a list of supported client authentication methods. The Token Endpoint MUST support *attest_jwt_client_auth* as defined in `[draft-ietf-oauth-attestation-based-client-auth-03] <https://www.ietf.org/archive/id/draft-ietf-oauth-attestation-based-client-auth-03.html#name-registration-of-attest_jwt_>`_.
+  * - **token_endpoint_auth_signing_alg_values_supported**
+    - JSON array containing a list of the JWS signing algorithms ("*alg*" values) supported by the token endpoint for the signature on the JWT used to authenticate the client at the Token Endpoint. See :rfc:`8414#section-2`.
+  * - **request_object_signing_alg_values_supported**
+    - JSON array containing a list of the JWS signing algorithms ("*alg*" values) supported for Request Objects. See `[openid-connect-discovery-1_0] <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata>`_.
+  * - **jwks**
+    - JSON Web Key Set document, passed by value, containing the protocol specific keys for the authorization server. See `OIDC-FED`_ Draft 36 Section 5.2.1 and `JWK`_.
+
+Metadata for openid_credential_issuer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The *openid_credential_issuer* metadata MUST contain the following claims.
+
+.. list-table:: 
+  :widths: 20 60
+  :header-rows: 1
+
+  * - **Claim**
+    - **Description**
+  * - **credential_issuer**
```

Comment:
authorization_servers is conditional required here, because VCI may be a different entity than the AS

https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.2.3

Reply:
I referred to this comment in a new issue (#333) as we currently don't support this feature in the current implementation